source/opt/optimizer.cpp

```diff
@@ -1034,6 +1034,11 @@ Optimizer::PassToken CreateEliminateDeadOutputStoresPass(
       MakeUnique<opt::EliminateDeadOutputStoresPass>(live_locs, live_builtins));
 }
 
+Optimizer::PassToken CreateEliminateDeadOutputComponentsPass() {
+  return MakeUnique<Optimizer::PassToken::Impl>(
+      MakeUnique<opt::EliminateDeadInputComponentsPass>(true));
```

Comment:
`/* output_instead */ true` if you keep this as-is.

Reply:
OK, since I am keeping as-is for the moment.